extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -823,22 +837,49 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
-   * Render creative via SafeFrame.
-   * @param {!ArrayBuffer} creativeBody  The creative, as raw bytes.
+   * Render the creative via some "cross domain iframe that accepts the creative
+   * in the name attribute".  This could be SafeFrame or the AMP-native
+   * NameFrame.
+   *
+   * @param {!ArrayBuffer} creativeBody
    * @return {!Promise} awaiting load event for ad frame
    * @private
    */
-  renderViaSafeFrame_(creativeBody) {
+  renderViaNameAttrOfXDomIframe_(creativeBody) {
```

Comment:
Sadly, it doesn't just do NameFrame yet.  At the moment, it does both NameFrame and SafeFrame.  This name is fugly, but captures both.  Happy to change it in the future, when we remove SafeFrame support.